robot/Cumulus/resources/NPSP.robot

```diff
@@ -430,10 +431,8 @@ Enable RD2QA
 
 Enable RD2
     [Documentation]           Checks if Rd2 settings are already enabled and then run the scripts to enable RD2
-    Go To Page                Custom         NPSP_Settings
-    Open Main Menu            Recurring Donations
-    ${rd2_enabled} =          Check Rd2 Is Enabled
-    Run Keyword if            "${rd2_enabled}"!="True"
+    ${ns} =                   Get NPSP Namespace Prefix
+    Run Keyword if            "${ns}"!="npsp__"
     ...                       Enable RD2QA
```

Comment:
so RD2 is enabled by default in regression org? Also since u have this logic in each test if u remove the previous check then in qa_org flow it would try to enable for every single test, is that what u intend to do?

Reply:
RD2 is not enabled on regression org by default, the discussion I had with Rick is they will enable it by running a different flow on the regression org after running all the other tests first and then execute these tests. I tested accordingly.
Yes, I changed the logic to enable it for each test. That way any test can be run individually.

Reply:
spoke through slack to get more clarity.